tests/tools/oesign/test-enclave/host/host.c

```diff
@@ -93,7 +94,7 @@ int main(int argc, const char* argv[])
         }
     }
 
-    if (_is_kss_supported())
```

Comment:
This works to unblock, but ideally test could work without the quote provider; `check_kss_extended_ids` is making the call with `OE_REPORT_FLAGS_REMOTE_ATTESTATION` but it could just get the local report and validate the IDs from that directly.

Reply:
@anakrish We could approve this and follow up with a more comprehensive fix. Thoughts?

Reply:
cc @mingweishih

Reply:
cc @manojrupireddy